tsl/test/expected/dist_compression.out

```diff
@@ -510,3 +552,151 @@ hypertable_id|attname |compression_algorithm_id|segmentby_column_index|orderby_c
+(1 row)
+
+--TEST 1--
+--cannot set policy without enabling compression --
+\set ON_ERROR_STOP 0
+select add_compression_policy('conditions', '60d'::interval);
+ERROR:  compression not enabled on hypertable "conditions"
+\set ON_ERROR_STOP 1
+-- TEST2 --
+--add a policy to compress chunks --
+alter table conditions set (timescaledb.compress, timescaledb.compress_segmentby = 'location', timescaledb.compress_orderby = 'time');
+insert into conditions
+select generate_series('2018-12-01 00:00'::timestamp, '2018-12-31 00:00'::timestamp, '1 day'), 'POR', 'klick', 55, 75;
+select add_compression_policy('conditions', '60d'::interval) AS compressjob_id
+\gset
+select * from _timescaledb_config.bgw_job where id = :compressjob_id;
+  id  |     application_name      | schedule_interval  | max_runtime | max_retries | retry_period |      proc_schema      |     proc_name      |    owner    | scheduled | hypertable_id |                       config                        
+------+---------------------------+--------------------+-------------+-------------+--------------+-----------------------+--------------------+-------------+-----------+---------------+-----------------------------------------------------
+ 1000 | Compression Policy [1000] | @ 15 days 12 hours | @ 0         |          -1 | @ 1 hour     | _timescaledb_internal | policy_compression | test_role_1 | t         |             2 | {"hypertable_id": 2, "compress_after": "@ 60 days"}
+(1 row)
+
+select * from alter_job(:compressjob_id, schedule_interval=>'1s');
+ job_id | schedule_interval | max_runtime | max_retries | retry_period | scheduled |                       config                        | next_start 
+--------+-------------------+-------------+-------------+--------------+-----------+-----------------------------------------------------+------------
+   1000 | @ 1 sec           | @ 0         |          -1 | @ 1 hour     | t         | {"hypertable_id": 2, "compress_after": "@ 60 days"} | -infinity
+(1 row)
+
+select * from _timescaledb_config.bgw_job where id >= 1000 ORDER BY id;
+  id  |     application_name      | schedule_interval | max_runtime | max_retries | retry_period |      proc_schema      |     proc_name      |    owner    | scheduled | hypertable_id |                       config                        
+------+---------------------------+-------------------+-------------+-------------+--------------+-----------------------+--------------------+-------------+-----------+---------------+-----------------------------------------------------
+ 1000 | Compression Policy [1000] | @ 1 sec           | @ 0         |          -1 | @ 1 hour     | _timescaledb_internal | policy_compression | test_role_1 | t         |             2 | {"hypertable_id": 2, "compress_after": "@ 60 days"}
+(1 row)
+
+insert into conditions
+select now()::timestamp, 'TOK', 'sony', 55, 75;
+-- TEST3 --
+--only the old chunks will get compressed when policy is executed--
+CALL run_job(:compressjob_id);
```

Comment:
should check status from _timescaledb_catalog.chunk here. (I didn't see a check for the catalog info. We should have atleast 1 test for it  for the distributed case and one for the non-distributed case since this code changes some of the semantics there).